core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationItem.java

```diff
@@ -0,0 +1,40 @@
+package org.arquillian.smart.testing.configuration;
+
+public class ConfigurationItem {
+
+    private final String paramName;
+    private String systemProperty;
+    private Object defaultValue;
+
+    public ConfigurationItem(String paramName, String systemProperty, Object defaultValue) {
```

Comment:
the constructors would deserve Javadoc - mainly describing the `paramName` - that it needs to match the set methods